airbyte-webapp/src/views/Connector/ServiceForm/components/Controls/ConnectorServiceTypeControl.tsx

```diff
@@ -187,6 +188,14 @@ const ConnectorServiceTypeControl: React.FC<{
           "9da77001-af33-4bcd-be46-6252bf9342b9", // Shopify
         ]
       : [];
+  if (
+    window.CLOUD === "true" &&
+    workspace.workspaceId === "54135667-ce73-4820-a93c-29fe1510d348"
+  ) {
+    disallowedOauthConnectors = disallowedOauthConnectors.filter(
+      (id) => id !== "9da77001-af33-4bcd-be46-6252bf9342b9"
+    );
+  }
```

Comment:
I think it would be easier if we're just adding the check in the above array instead of having another `if` clause here to filter it out again.

Common pattern to do this is:

```ts
window.CLOUD === "true"
  ? [
           "200330b2-ea62-4d11-ac6d-cfe3e3f8ab2b", // Snapchat
          "2470e835-feaf-4db6-96f3-70fd645acc77", // Salesforce Singer
          ...(workspace.workspaceId !== "..." ? ["9da77001-af33-4bcd-be46-6252bf9342b9"] : []), // Shopify
  ]
  : [];
```

Reply:
Also please add an inline comment above this place what workspace that is and why we're doing it, so it's instantly understandable from looking at the code why we're doing this.